### PR TITLE
TODO with OLS-619

### PR DESCRIPTION
**Pull request by tisnik**
## Description

TODO with [OLS-619](https://issues.redhat.com//browse/OLS-619)

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [x] Other

### CODE REVIEW

File: scripts/questions_gen.py [hunk 1]
```diff
@@ -245,6 +245,9 @@ async def questions_eval(
         res_row["passing"] = result.passing
         res_row["feedback"] = result.feedback
         res_row["score"] = result.score
+
+        # TODO: OLS-619 Evaluation model can return `None` instead of score, so
+        #               it is not possible to convert the value w/o checks
```
Comment (onmete) [inline scripts/questions_gen.py:250]: indentation is off

Comment (tisnik) [reply scripts/questions_gen.py:250]: no, I wanted to format the text into paragraph, it's on purpose

Comment (onmete) [reply scripts/questions_gen.py:250]: hm, ok

**Pull request merged**